src/time/BrokenDate.hpp

```diff
@@ -60,6 +63,8 @@ struct BrokenDate {
    */
   BrokenDate() = default;
 
+  BrokenDate(const TCHAR* date_string);
```

Comment:
Undocumented. What input does it accept? What if an error occurs?

Reply:
removed..